web-server/v0.4/src/common/menu.js

```diff
@@ -5,6 +5,18 @@ const menuData = [
     name: 'Dashboard',
     icon: 'dashboard',
     path: '/',
+    routes: [
```

Comment:
This is a great start, we'll also want to add routes for the remaining pages in the dashboard:

```javascript
{
  name: 'Results',
  path: '/results',
  routes: [
    {
      name: 'Summary',
      path: '/summary',
    },
    {
      name: 'Comparison Select',
      path: '/comparison-select',
      routes: [
        {
          name: 'Comparison',
          path: '/comparison'
        }
      ]
    }
  ],
}
```